docroot/profiles/custom/cgov_site/modules/custom/cgov_home_landing/cgov_home_landing.module

```diff
@@ -89,3 +91,28 @@ function cgov_home_landing_form_node_form_alter(&$form, \Drupal\Core\Form\FormSt
 
   return $form;
 }
+
+/**
+ * Implements hook_ENTITY_TYPE_view().
+ *
+ * This changes the view mode of the image for
+ * ncids_image so it renders with the proper aspect ratio.
+ */
+function cgov_home_landing_paragraph_view(array &$build, ParagraphInterface $entity, EntityViewDisplayInterface $display, $view_mode) {
+  $bundle = $entity->bundle();
+  if ($bundle === 'ncids_image') {
+    $crop_display = $entity->field_image_aspect_ratio->value;
```

Comment:
See the note below about the change to either the field or the values.

Reply:
I changed the name to `field_ncids_img_para_ratio` so that we know it is specifically for the NCIDS Image paragraph.